package.json

```diff
@@ -44,7 +44,7 @@
     "dotenv": "^10.0.0",
     "eslint": "^6.7.2",
     "eslint-plugin-vue": "^6.2.2",
-    "node-sass": "^4.14.1",
+    "sass": "^4.14.1",
```

Comment:
Als ik nu een `yarn (install)` doe dan krijg ik de vraag welke versie ik wil installeren. Zo te zien is 1.43.4 de laatste versie dus ik zou er `"sass": ^1.43.0` van maken.